requirements.txt

```diff
@@ -1,7 +1,8 @@
 Flask==0.12.0
-earthengine-api==0.1.102
+earthengine-api==0.1.140
 CTRegisterMicroserviceFlask==0.4.0
 pycrypto==2.6.1
 requests==2.12.4
 sql2gee==0.2.5
-oauth2client==4.1.2
+google-api-python-client==1.7.3
+oauth2client==4.1.3
```

Comment:
this is missing. Somehow, docker installs this, but if you run the app natively, you'll need it.